packages/frontend/src/components/login/v2/ConfirmLoginWrapper.js

```diff
@@ -33,20 +32,7 @@ export default ({
             publicKey={publicKey}
             contractId={contractId}
             onClickCancel={onClickCancel}
-            onClickConnect={async () => {
-                await Mixpanel.withTracking('LOGIN',
-                    async () => {
-                        await dispatch(allowLogin());
-                    },
-                    (e) => {
-                        dispatch(showCustomAlert({
-                            success: false,
-                            messageCodeHeader: 'error',
-                            errorMessage: e.message
-                        }));
```

Comment:
`allowLogin` already handles populating the `alert` `meta` so this actually just produces a duplicate alert.